desktop/components/auth_modal/view.coffee

```diff
@@ -44,12 +44,19 @@ module.exports = class AuthModalView extends ModalView
     mode = mode: options.mode if options.mode
     @state = new State mode
 
+    currentPath = location.pathname
+    postLoginPath = @redirectTo or currentPath
+    postSignupPath = @redirectTo or '/personalize'
+    if (options.mode is 'signup' or options.mode is 'register') and !@destination
+      @destination = currentPath unless @redirectTo
```

Comment:
I think this default makes sense. If you're signing up and pass in a `redirectTo`, then you're skipping onboarding altogether and so `destination` doesnt need to be set. And, if you're explicitly passing in a post onboarding `destination`, we should honor that.

But otherwise, just bring the user back to where they started.